### PR TITLE
[WIP] Renovation: allow render button in different containers

### PR DESCRIPTION
**Pull request by LazyLahtak**

### CODE REVIEW

File: js/renovation/widget.tsx [hunk 1]
```diff
@@ -119,6 +119,7 @@ export class WidgetInput {
     @Event() onKeyboardHandled?: (args: any) => any | undefined;
     @Event() onKeyPress?: (e: any, options: any) => any;
     @Event() onVisibilityChange?: (args: boolean) => undefined;
+    @OneWay() rootNode?: any = 'div';
```
Comment (roman-simionov) [inline js/renovation/widget.tsx:122]: type should be a string

Comment (LazyLahtak) [reply js/renovation/widget.tsx:122]: If I set type `string`, we get an error:
`JSX element type 'viewModel.props.rootNode' does not have any construct or call signatures`

Comment (LazyLahtak) [reply js/renovation/widget.tsx:122]: If I pass render function instead string:
```jsx
rootNode === (props) => Preact.h(nodeName, props)
```
then, we can't use JSX (generator doesn't recognise correctrly)

Comment (roman-simionov) [reply js/renovation/widget.tsx:122]: ok

**Pull request closed without merging**